### DISCUSSION TITLE
OSD Scale for volume setting does not affect non-DDC audio volume

### DISCUSSION THREAD

**Discussion by shubh30112003** [MonitorControl/MonitorControl#1574] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

osd scale for volume control doesnt work. i tried on sonoma as well as os 15. 

### Steps to reproduce

turn on osd scale and try 

### Expected behavior

its should work like brightness osd scale. increase brightness by bits

### Anything else?

<img width="842" alt="Screenshot 2024-06-15 at 1 21 57 PM" src="https://github.com/MonitorControl/MonitorControl/assets/121214849/de8c78da-38c9-466f-b552-09962fd078a7">

https://github.com/MonitorControl/MonitorControl/assets/121214849/9ca24488-18fc-4c62-b769-1a9574678f61

### Environment Information (please complete the following information)

```markdown
- macOS version: Version 15.0 Beta (24A5264n)
- Mac model: MacBook Pro,2021 Apple M1 Pro
- MonitorControl version: 4.3.0 beta
- Monitor(s):0
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment, marked as answer]
Hi there - this setting only affects external displays with DDC volume control - does not affect the other audio devices, like the built-in speaker or other sound outputs.
